lib/runner.js

```diff
@@ -272,7 +272,11 @@ Runner.prototype.failHook = function(hook, err) {
     hook.title =
       hook.originalTitle + ' for "' + hook.ctx.currentTest.title + '"';
   } else {
-    var parent = hook.parent ? hook.parent.title : 'root';
+    var parent = hook.parent.title
+      ? hook.parent.title
+      : hook.parent.root && !hook.parent.parent
+        ? 'root'
+        : '';
```

Comment:
`hook.parent.root` is not reliable since any suite with an empty string title "" is marked as root. That's why I'm testing for `hook.parent.parent`, also

Reply:
Sounds like YA bug...

Reply:
> `hook.parent.root` is not reliable since any suite with an empty string title "" is marked as root.

Sounds like YA bug...

Reply:
see PR #3632

Reply:
Should the root suite be output as maybe `'{root}'` instead to designate its implied title (rather than an explicit one).